internal/config/config.go

```diff
@@ -152,7 +152,7 @@ func WithDefaults() *Config {
 		PingTimeout:               15 * time.Second,
 		CanUseMultipleCatalogs:    true,
 		DriverName:                "godatabrickssqlconnector", // important. Do not change
-		DriverVersion:             "0.9.0",
+		DriverVersion:             "1.0.1",
```

Comment:
maybe the best way is to have a version.go file at top level folder instead of the config? It is pretty hidden in here :-)

Reply:
How about a `constants.go` file? Then we can have DriverVersion, DriverName as constants in the top level folder and refer to them here?